(docs) - quick-start (#665)
FormidableLabs/urql: docs/overview.md

---
title: Overview
order: 1
---

# Overview

`urql` is an implementation of a GraphQL client, built to be both easy to use for newcomers to
GraphQL, as well as extensible, to grow to support dynamic single-app applications and highly
customized GraphQL infrastructure. In short, `urql` prioritizes usability and adaptability.

As you're adopting GraphQL, `urql` becomes your primary data layer and can handle content-heavy
pages through ["Document Caching"](./basics/document-caching.md) as well as dynamic and data-heavy
apps through ["Normalized Caching"](./graphcache/normalized-caching.md).

## Constituent Parts

> `urql` can be understood as a collection of connected parts and packages. When you [first get
> started](./basics/getting-started.md) you'll only need to install a single package for your
> framework of choice. We're then able to declaratively send GraphQL requests to our API.

Reviewer: Nit: I think this mix between `you` and `we` is a tiny bit strange in this paragraph. Maybe we could make it all third person?

```suggestion
`urql` can be understood as a collection of connected parts and packages. When [getting started](./basics/getting-started.md) we only need to install a single package for our
framework of choice. We're then able to declaratively send GraphQL requests to our API.
```
